docs/lite.md

```diff
@@ -17,13 +17,13 @@ The Pluggable Modular Parsers provide the ability to customize and extend the da
 The Splunk platform is the destination for the syslog data. Splunk allows for comprehensive analysis, searching, and visualization of the processed data.
 
 ##  Data Flow
-Source systems send syslog data to SC4S Lite. The data may be transmitted via UDP, TCP, or RELP, depending on the system's capabilities and configurations.
+Source systems send syslog data to SC4S Lite. The data may be transmitted via UDP or TCP depending on the system's capabilities and configurations.
 SC4S Lite receives the syslog data and routes it through the appropriate parsers, as defined by user configurations.
 The parsers in pluggable module process the data, including parsing, filtering, and enriching the data with metadata.
 SC4S Lite forwards the processed syslog data to the Splunk platform over the HTTP Event Collector (HEC).
 
 ## Security Considerations
-SC4S Lite is built on an alpine lightweight container which has very little vulnerability as well as it  supports secure syslog data transmission protocols (such as RELP and TLS over TCP) to protect the data in transit. Furthermore, the environment in which SC4S Lite is deployed enhances data security.
+SC4S Lite is built on an alpine lightweight container which has very little vulnerability as well as it  supports secure syslog data transmission protocols (such as TLS over TCP) to protect the data in transit. Furthermore, the environment in which SC4S Lite is deployed enhances data security.
 
 ## Scalability and Performance
```

Comment:
SC4S Lite provides superior performance and scalability by implementing lightweight architecture and pluggable parsers that distribute the processing load. It is also packaged with an eBPF program to help further enhance performance.